tutorials/mongoose/deleteById.md

```diff
@@ -0,0 +1,21 @@
+There is currently no method called `deleteById()` in mongoose.
+However, there is the `deleteOne()` method with takes a parameter, `filter`, which indicates what document to delete.
+Simply pass the `_id` as the `filter` and the document will be deleted.
+
+```javascript
+const testSchema = new mongoose.Schema({
+    name: String
+});
+
+const Test = mongoose.model('Test', testSchema);
+
+
+async function run() {
+const entry = await Test.create({ name: 'Masteringjs.io' });
+await Test.find();
+await Test.deleteOne({_id: entry._id});
+await Test.find();
+}
+
+run();
+```
```

Comment:
Also add an example of defining a `deleteById()` [instance method](https://mongoosejs.com/docs/guide.html#methods)

Reply:
instance method or static function? From what I read, it seems that `instance methods` are a per document usage while `statics` are for the whole model.